src/main/java/seedu/address/model/module/Module.java

```diff
@@ -35,8 +29,6 @@ public Module(Code code, Name name, String gradeDist) {
         this.code = code;
         this.title = name;
         this.gradeDist = gradeDist;
-        this.lessons = new ArrayList<>();
-        this.tasks = new ArrayList<>();
```

Comment:
Can I check why this is removed? Are we leaving it to later to implement?

Reply:
When I thought on how to handle the program's logic, I found out that putting lessons and tasks in Module will make it way harder to find/delete tasks & lessons. Instead, I think that we just keep lessons and tasks in separate lists and everytime we need to find lessons or tasks of a module, we can just do a O(n) search. This will make the program's logic much simpler and less prone to errors